src/reason_parser_message.ml

```diff
@@ -10,29 +10,29 @@ let message =
         "<SYNTAX ERROR>\n"
```

Comment:
Not related to this diff: 

The diff on this file is getting more annoying. I think we should just ignore this file and generate it on the fly.